libs/data-access-api/src/lib/data-access-api/recipes.repository.ts

```diff
@@ -0,0 +1,17 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { RecipeDto } from './dto/RecipeDto';
+
+@Injectable()
+export class RecipesRepository {
+  private baseUrl: string;
+
+  constructor(private client: HttpClient) {
+    this.baseUrl = `/api/recipes`;
+  }
```

Comment:
give the new `inject` function a try
```suggestion
  private baseUrl = `/api/recipes`;
  private client = inject(HttpClient);
```